src/pdf.js

```diff
@@ -123,6 +124,7 @@ export {
   getDocument,
   getFilenameFromUrl,
   getPdfFilenameFromUrl,
+  getRGB,
```

Comment:
Why is this necessary, since I'm not finding anything that depends on this?
As a general rule, please let's not expose things *in the API* unless it's actually needed e.g. in the viewer.